### DISCUSSION TITLE
Development

### DISCUSSION THREAD

**Discussion by JuicyNuts** [SmartlyDressedGames/Unturned-3.x-Community#4724] [General]
Was wondering about why Nelson doesn't hire a development team for unturned? As you may have seen, unturned has started to lose its players and I wouldn't necessarily blame Nelson for it. But, why not have a development team you boss around and tell them what to do new mechanics and updates would be implemented into the game? I understand some people play on potatoes, but having a graphics update wouldn't hurt anyone, could just turn it off in the settings ig, but unturned needs new things, maybe instead of having unturned 2, just implement most ideas from there into Unturned, instead of having an entirely new game which I know for a fact if it ever gets released, players won't be switching to unturned 2, so the best bet is to implement things from unturned 2 to unturned.

**Comment by SDGNelson** [comment]
Copying from a similar [comment](https://steamcommunity.com/app/304930/discussions/0/4852155593256467723/#c4852155797906176304) I wrote recently on Steam discussions:

Player-count-wise it's taken longer than I would have guessed after turning off the playtime drops, but judging from the drop I think/hope any item farmers have left. It's quite similar to the 2015 era of the game which was a fun time, and there's a lot of great fun stuff going on in the community now, too.

I think I have a lot to learn when it comes to collaborative gamedev. I also don't want to become a manager with little time spent developing games anymore, I want to be spending most of my time doing gamedev - I already find it tricky to balance my non-gamedev responsibilites like customer support and the business end of things. That said, I agree working with talented modders for the game is extremely beneficial, and we've been experimenting with that a bit more alongside the existing collaborations on curated maps.

**Comment by JuicyNuts** [comment]
Well, that's some good news. Its not really making you a manager its more of like people helping with certain aspects of the game so the load isn't fully on you. Although I applaud you for all the work you've done especially when you are a lone developer and I would love to see how unturned changes in the future. 

As well as, would love to see stuff that was in Unturned 2 get added into Unturned.
